sample-lnd.conf

```diff
@@ -270,6 +270,15 @@
 ; 65536. The profile can be access at: http://localhost:<PORT>/debug/pprof/.
 ; profile=
 
+; Enable a blocking profile to be obtained from the profiling port. A blocking
+; profile can show where goroutines are blocking (stuck on mutexes, I/O, etc)
+; blockingprofile=1
```

Comment:
looks like we need to set default values here to pass the `sample configuration check`?